fairscale/experimental/wgit/signal_sparsity.py

```diff
@@ -40,7 +120,7 @@ class SignalSparsity:
         dst_top_k_element (int, optional):
             Number of top-k elements to retain for DST. Default: None
         dst_top_k_percent (float, optional):
-            Percent of top-k elements to retain for DST. Default: 0.1
+            Percent of top-k elements to retain for DST. Default: None
```

Comment:
line 129 the example need to be updated too? Otherwise, that will be incorrect example?

Reply:
Thanks for pointing out!